.github/workflows/go.yml

```diff
@@ -0,0 +1,28 @@
+# This workflow will build a golang project
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-go
+
+name: Go
```

Comment:
Small detail but, is it possible to name the two checks something more meaningful for devs to understand what is happening. Ex step 1: Compose Setup, step 2: Build & Test?

Reply:
Can do!